src/test/java/page/HomePage.java

```diff
@@ -0,0 +1,57 @@
+package page;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class HomePage extends BasePage{
+
+    By avatarIcon = By.cssSelector("img.avatar");
+    By playlistEditField = By.xpath("//input[@data-testid=\"inline-playlist-name-input\"]");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```

Reply:
Thank You. For some reason all msgs were going to my Spam. I only noticed when I got the msg here on InteliJ. I am going to Fix and Upload and go through all the other branches as well. 
Thank you again!